docs/adr/0011-message-timing-information.md

```diff
@@ -23,30 +24,29 @@ timestamps of interest, and the decisions made in each case.
 
 ### Command Messages
 
-We believe the initial requirement that the application "model time" is still
+We believe the existing requirement that the application "model time" is still
 appropriate for command messages. The time at which the command message is
-created is irrelevant; any time information relevant to the domain logic should
-be included in the message itself.
+created or enqueued is irrelevant; any time information relevant to the domain
+logic should be included in the message itself.
 
 **We have decided not to expose the command creation time.**
 
 ### Event Messages
 
 The time at which an event is recorded is a fundamental property of the event
 itself. Put another way, every event occurs at some time regardless of whether
-the domain has any time-based logic.
+the domain is time-based.
 
-Furthermore, the time at which event occurs may be relevant to some domain logic
-that is *triggered* by the event, even if the aggregate that *produced* the event
-has no time-based logic.
+Furthermore, the time at which event occurs may be relevant to some ancillary
```

Comment:
This might be better as *an event* or *the event*?
> the time at which event occurs may

Reply:
Added missing `the`.